### PR TITLE
[DE] Add `meta/flag-references` and update flag titles in German articles

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: meta/flag-references/de.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+[flag_BS]: /wiki/shared/flag/BS.gif "Bahamas"
+[flag_BT]: /wiki/shared/flag/BT.gif "Bhutan"
+[flag_BW]: /wiki/shared/flag/BW.gif "Botswana"
+[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
+[flag_BZ]: /wiki/shared/flag/BZ.gif "Belize"
+[flag_CA]: /wiki/shared/flag/CA.gif "Kanada"
+[flag_CD]: /wiki/shared/flag/CD.gif "Demokratische Republik Kongo"
+[flag_CF]: /wiki/shared/flag/CF.gif "Zentralafrikanische Republik"
+[flag_CG]: /wiki/shared/flag/CG.gif "Kongo"
+[flag_CH]: /wiki/shared/flag/CH.gif "Schweitz"
```
Comment (Phippe) [inline meta/flag-references/de.md:41]: Don't do this to me :')

File: meta/flag-references/de.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+[flag_ET]: /wiki/shared/flag/ET.gif "Äthiopien"
+[flag_EU]: /wiki/shared/flag/EU.gif "Europäische Union"
+[flag_FI]: /wiki/shared/flag/FI.gif "Finnland"
+[flag_FJ]: /wiki/shared/flag/FJ.gif "Fidschi"
+[flag_FK]: /wiki/shared/flag/FK.gif "Falklandinseln (Malvinas)"
+[flag_FM]: /wiki/shared/flag/FM.gif "Föderierte Staaten von Mikronesien"
+[flag_FO]: /wiki/shared/flag/FO.gif "Färöer"
+[flag_FR]: /wiki/shared/flag/FR.gif "Frankreich"
+[flag_GA]: /wiki/shared/flag/GA.gif "Gabun"
+[flag_GB]: /wiki/shared/flag/GB.gif "Vereignigtes Königreich"
```
Comment (Phippe) [inline meta/flag-references/de.md:73]:
```suggestion
[flag_GB]: /wiki/shared/flag/GB.gif "Vereinigtes Königreich"
```

**Review by Phippe — CHANGES_REQUESTED** [review]
The countries I know look correct apart from those two

File: meta/flag-references/de.md (hunk 1 above)
Comment (TicClick) [reply meta/flag-references/de.md:41]: +1

Comment (The-Last-Cookie) [reply meta/flag-references/de.md:41]: ~~I just wanted to know if you actually review this 👀 ~~

File: meta/flag-references/de.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+[flag_AD]: /wiki/shared/flag/AD.gif "Andorra"
+[flag_AE]: /wiki/shared/flag/AE.gif "Vereinigte Arabische Emirate"
+[flag_AF]: /wiki/shared/flag/AF.gif "Afghanistan"
+[flag_AG]: /wiki/shared/flag/AG.gif "Antigua und Barbuda"
+[flag_AI]: /wiki/shared/flag/AI.gif "Anguilla"
+[flag_AL]: /wiki/shared/flag/AL.gif "Albanien"
+[flag_AM]: /wiki/shared/flag/AM.gif "Armenien"
+[flag_AN]: /wiki/shared/flag/AN.gif "Niederländische Antillen"
+[flag_AO]: /wiki/shared/flag/AO.gif "Angola"
+[flag_AR]: /wiki/shared/flag/AR.gif "Argentinen"
```
Comment (respektive) [inline meta/flag-references/de.md:11]:
```suggestion
[flag_AR]: /wiki/shared/flag/AR.gif "Argentinien"
```

File: meta/flag-references/de.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+[flag_AS]: /wiki/shared/flag/AS.gif "Amerikanisch-Samoa"
+[flag_AT]: /wiki/shared/flag/AT.gif "Österreich"
+[flag_AU]: /wiki/shared/flag/AU.gif "Australien"
+[flag_AW]: /wiki/shared/flag/AW.gif "Aruba"
+[flag_AX]: /wiki/shared/flag/AX.gif "Aland"
```
Comment (respektive) [inline meta/flag-references/de.md:16]:
```suggestion
[flag_AX]: /wiki/shared/flag/AX.gif "Åland"
```

File: meta/flag-references/de.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+[flag_AZ]: /wiki/shared/flag/AZ.gif "Azerbaidschan"
```
Comment (respektive) [inline meta/flag-references/de.md:17]:
```suggestion
[flag_AZ]: /wiki/shared/flag/AZ.gif "Aserbaidschan"
```

File: meta/flag-references/de.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+[flag_BA]: /wiki/shared/flag/BA.gif "Bosnien und Herzegowina"
+[flag_BB]: /wiki/shared/flag/BB.gif "Barbados"
+[flag_BD]: /wiki/shared/flag/BD.gif "Bangladesch"
+[flag_BE]: /wiki/shared/flag/BE.gif "Belgien"
+[flag_BF]: /wiki/shared/flag/BF.gif "Burkina Faso"
+[flag_BG]: /wiki/shared/flag/BG.gif "Bulgarien"
+[flag_BH]: /wiki/shared/flag/BH.gif "Bahrain"
+[flag_BI]: /wiki/shared/flag/BI.gif "Burundi"
+[flag_BJ]: /wiki/shared/flag/BJ.gif "Benin"
+[flag_BL]: /wiki/shared/flag/BL.gif "Saint-Barthelemy"
```
Comment (respektive) [inline meta/flag-references/de.md:27]:
```suggestion
[flag_BL]: /wiki/shared/flag/BL.gif "Saint-Barthélemy"
```

File: meta/flag-references/de.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+[flag_EC]: /wiki/shared/flag/EC.gif "Ecuador"
+[flag_EE]: /wiki/shared/flag/EE.gif "Estland"
+[flag_EG]: /wiki/shared/flag/EG.gif "Ägypten"
+[flag_ER]: /wiki/shared/flag/ER.gif "Eritrea"
+[flag_ES]: /wiki/shared/flag/ES.gif "Spanien"
+[flag_ET]: /wiki/shared/flag/ET.gif "Äthiopien"
+[flag_EU]: /wiki/shared/flag/EU.gif "Europäische Union"
+[flag_FI]: /wiki/shared/flag/FI.gif "Finnland"
+[flag_FJ]: /wiki/shared/flag/FJ.gif "Fidschi"
+[flag_FK]: /wiki/shared/flag/FK.gif "Falklandinseln (Malvinas)"
```
Comment (respektive) [inline meta/flag-references/de.md:68]:
```suggestion
[flag_FK]: /wiki/shared/flag/FK.gif "Falklandinseln (Malvinen)"
```

File: meta/flag-references/de.md [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+[flag_RW]: /wiki/shared/flag/RW.gif "Ruanda"
+[flag_SA]: /wiki/shared/flag/SA.gif "Saudi-Arabien"
+[flag_SB]: /wiki/shared/flag/SB.gif "Salomonen"
+[flag_SC]: /wiki/shared/flag/SC.gif "Seychellen"
+[flag_SD]: /wiki/shared/flag/SD.gif "Sudan"
+[flag_SE]: /wiki/shared/flag/SE.gif "Schweden"
+[flag_SG]: /wiki/shared/flag/SG.gif "Singapur"
+[flag_SH]: /wiki/shared/flag/SH.gif "St. Helena"
+[flag_SI]: /wiki/shared/flag/SI.gif "Slowenien"
+[flag_SJ]: /wiki/shared/flag/SJ.gif "Svalbard und Jan Mayen"
```
Comment (respektive) [inline meta/flag-references/de.md:195]:
```suggestion
[flag_SJ]: /wiki/shared/flag/SJ.gif "Spitzbergen und Jan Mayen"
```
apparently both is fine? taking [this](https://de.wikipedia.org/wiki/ISO-3166-1-Kodierliste) as reference but when you click on the `SJ` link its written like yours so idk lol

File: meta/flag-references/de.md [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+[flag_SK]: /wiki/shared/flag/SK.gif "Slowakei"
+[flag_SL]: /wiki/shared/flag/SL.gif "Sierra Leone"
+[flag_SM]: /wiki/shared/flag/SM.gif "San Marino"
+[flag_SN]: /wiki/shared/flag/SN.gif "Senegal"
+[flag_SO]: /wiki/shared/flag/SO.gif "Somalia"
+[flag_SR]: /wiki/shared/flag/SR.gif "Suriname"
+[flag_ST]: /wiki/shared/flag/ST.gif "Sao Tome und Principe"
```
Comment (respektive) [inline meta/flag-references/de.md:202]:
```suggestion
[flag_ST]: /wiki/shared/flag/ST.gif "São Tomé und Príncipe"
```

File: meta/flag-references/de.md [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+[flag_SV]: /wiki/shared/flag/SV.gif "El Salvador"
+[flag_SY]: /wiki/shared/flag/SY.gif "Syrische Arabische Republik"
```
Comment (respektive) [inline meta/flag-references/de.md:204]:
```suggestion
[flag_SY]: /wiki/shared/flag/SY.gif "Syrien"
```
again both fine i think, but `Syrien` is more common

File: meta/flag-references/de.md [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+[flag_SZ]: /wiki/shared/flag/SZ.gif "Eswatini"
+[flag_TC]: /wiki/shared/flag/TC.gif "Turks- und Caicosinseln"
+[flag_TD]: /wiki/shared/flag/TD.gif "Tschad"
+[flag_TG]: /wiki/shared/flag/TG.gif "Togo"
+[flag_TH]: /wiki/shared/flag/TH.gif "Thailand"
+[flag_TJ]: /wiki/shared/flag/TJ.gif "Tadschikistan"
+[flag_TL]: /wiki/shared/flag/TL.gif "Timor-Leste"
```
Comment (respektive) [inline meta/flag-references/de.md:211]:
```suggestion
[flag_TL]: /wiki/shared/flag/TL.gif "Osttimor"
```

**Review by respektive — CHANGES_REQUESTED** [review]
[source](https://de.wikipedia.org/wiki/ISO-3166-1-Kodierliste)

**Comment by The-Last-Cookie** [comment]
I saw the different special characters, like í, ã etc. too. It wasn't included in the English original, so I didn't include it in the German one either.

**Comment by respektive** [comment]
> I saw the different special characters, like í, ã etc. too. It wasn't included in the English original, so I didn't include it in the German one either.

oh i didnt check that, i guess those are fine then

File: meta/flag-references/de.md (hunk 7 above)
Comment (The-Last-Cookie) [reply meta/flag-references/de.md:68]: [This source](https://www.garantiert-reisen.de/reiseinformationen-falklandinseln/) writes it `Malwinen`.

File: meta/flag-references/de.md (hunk 8 above)
Comment (The-Last-Cookie) [reply meta/flag-references/de.md:195]: `Svalbard` might be a more international term, so I will use this.

File: meta/flag-references/de.md (hunk 7 above)
Comment (respektive) [reply meta/flag-references/de.md:68]: oh yeah [wikipedia](https://de.wikipedia.org/wiki/Falklandinseln) does aswell. my mistake, i updated the suggestion

**Review by TicClick — APPROVED** [review]

**Pull request merged**